src/main/java/com/gluonhq/Main.java

```diff
@@ -115,6 +117,8 @@ public Double fromString(String s) {
                 fontSize,
                 actionButton(LineAwesomeSolid.BOLD, editor.getActionFactory().decorate(TextDecoration.builder().fontWeight(FontWeight.BOLD).build())),
                 actionButton(LineAwesomeSolid.ITALIC, editor.getActionFactory().decorate(TextDecoration.builder().fontPosture(FontPosture.ITALIC).build())),
+                actionToggleButton(LineAwesomeSolid.STRIKETHROUGH, a -> editor.getActionFactory().decorate(TextDecoration.builder().strikethrough(a).build())), 
```

Comment:
as discussed, until we support toggles properly, this should be just an actionButton? (Button/action should be disabled if there is no selection for now)